pkg/common/common.go

```diff
@@ -11,3 +22,34 @@ const (
 	PolicyViolation = "POLICYVIOLATION"
 	PolicyReport    = "POLICYREPORT"
 )
+
+func GetNamespaceSelectors(request *v1beta1.AdmissionRequest, resource *unstructured.Unstructured, nsLister listerv1.NamespaceLister) map[string]string {
```

Comment:
Can we refactor it to
`GetNamespaceSelectors(resource *unstructured.Unstructured, nsLister listerv1.NamespaceLister)`